conf/application.conf

```diff
@@ -48,6 +48,9 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
 # Provides an implementation and configures all filters required by a Platform microservice.
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.MicroserviceModule"
 
+# Starts up the UC THreshold Orchestrator
+play.modules.enabled += "uk.gov.hmrc.helptosave.modules.UCThresholdModule"
```

Comment:
do we have this class `UCThresholdModule ` anywhere I dont see it in the PR ?

Reply:
you're right, I can't see it either. We'll investigate...

Reply:
we've pushed a new commit with the missing stuff included. Turns out gitignore was ignoring the files